docs/basics/userguide/save_load.md

```diff
@@ -149,9 +149,130 @@ Success!
+env_seed_max_value = 500
+
+env_id = "FrozenLake-v1"  # Id of the environment
+
+env = gym_make(env_id)
+env.seed(int(seeder.rng.integers(env_seed_max_value)))
+agent_to_train_and_save = QLAgent(
+    env,
+    gamma=0.95,
+    alpha=0.8,
+    exploration_type="epsilon",
+    exploration_rate=0.25,
+    seeder=seeder,
+)
+agent_to_train_and_save.fit(300000)  # Agent's training
+print(agent_to_train_and_save.Q)  # print the content of the Q-table
+
+agent_to_train_and_save.save("./results/")  # save the agent
+```
+
+```none
+[INFO] 11:28:                                 agent_name  worker  episode_rewards  max_global_step
+                                                  QL        -1         0.0             195540
+[[0.1830874  0.15802259 0.12087594 0.16358512]
+ [0.         0.         0.         0.16674384]
+ [0.10049071 0.09517673 0.11326436 0.07236883]
+ [0.10552007 0.06660356 0.07020302 0.1104349 ]
+ [0.23065463 0.         0.19028937 0.20689438]
+ [0.         0.         0.         0.        ]
+ [0.08408004 0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.17382279 0.         0.2417443 ]
+ [0.         0.29498867 0.         0.        ]
+ [0.46487572 0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.52043878 0.56986596 0.19259904]
+ [0.57831479 0.6858159  0.22998936 0.39350426]
+ [0.         0.         0.         0.        ]]
+```
+
+### Load the agent
+
+To load an agent, you should use its `load()` function. But be careful, you have to add some not-saved parameters (in this case, the environment). This parameters should be given through a `dict`.
+
+
+```python
+# create a seeded env
+env_for_loader = gym_make(env_id)
+env_for_loader.seed(int(seeder.rng.integers(env_seed_max_value)))
+
+# create the 'not-saved parameters' dict
+params_for_loader = dict(env=env_for_loader)
+
+# load the agent
+loaded_agent = QLAgent.load("./results/", **params_for_loader)
+print(loaded_agent.Q)  # print the content of the Q-table
+```
+
+
+```none
+[[0.1830874  0.15802259 0.12087594 0.16358512]
+ [0.         0.         0.         0.16674384]
+ [0.10049071 0.09517673 0.11326436 0.07236883]
+ [0.10552007 0.06660356 0.07020302 0.1104349 ]
+ [0.23065463 0.         0.19028937 0.20689438]
+ [0.         0.         0.         0.        ]
+ [0.08408004 0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.17382279 0.         0.2417443 ]
+ [0.         0.29498867 0.         0.        ]
+ [0.46487572 0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.         0.         0.        ]
+ [0.         0.52043878 0.56986596 0.19259904]
+ [0.57831479 0.6858159  0.22998936 0.39350426]
+ [0.         0.         0.         0.        ]]
+```
+
+As you can see, we haven't re-fit the agent, and the q-table is the same as the one previously saved.
```

Comment:
Not really useful is it?

Reply:
Just to prove we've loaded what we've saved